modules/nf-core/wittyer/main.nf

```diff
@@ -0,0 +1,69 @@
+process WITTYER {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "quay.io/nf-core/wittier:0.3.3"
```

Comment:
Here I am unsure if that is the correct "address" to get the container.

Reply:
remove quay.io